scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -12,156 +12,260 @@ output:
 
-```{r}
-sample_metadata
+In the `reducedDim` slots you should see `PCA` and `UMAP` which both correspond to the pre-integrated dimensionality reduction results.
+You should also see `fastmnn_PCA` and `fastmnn_UMAP` reduced dimensions, which correspond to the integrated results.
+
+Just like in the integration notebook, this dataset also contains the cell type annotations found in the `celltype_fine` and `celltype_broad` columns of the `colData`.
+These cell types were originally assigned in [Patel _et al._ (2022)](https://doi.org/10.1016/j.devcel.2022.04.003).
+We will use these cell type assignments to set up the DE analysis below, but they are not required.
+It's important to note that DE analysis can be applied to any subpopulation of interest that is shared across samples and does not need to be explicitly defined by a cell type.
+
+Because we are going to be doing DE analysis between ARMS and ERMS samples, let's start by labeling cells in the integrated dataset based on their RMS subtype.
+To do this we will need to be sure that the subtype is present in the `colData` of the integrated SCE object.
+If it's not there, we need to add it in.
+
+```{r coldata head}
+# look at the head of the coldata
+head(colData(integrated_sce))
 ```
 
-Looking at this sample table, we notice that all the samples are of the same diagnosis, rhabdomyosarcoma, but come from two types of rhabdomyosarcoma - embryonal rhabdomyosarcoma (ERMS) and alveolar rhabdomyosarcoma (ARMS).
-These samples also come from a variety of tissue sites and stages of disease (initial diagnosis vs recurrence).
+Fortunately we also have the sample metadata table that we read in earlier, which contains information about each of the samples present in the dataset.
 
-For today's example, we are particularly interested in comparing the gene expression across cell types in samples from patients diagnosed with ARMS vs. samples from patients diagnosed with ERMS.
-To help us out later on when we are performing our analysis and making plots, let's incorporate the information in this sample metadata table into the `colData` of the integrated dataset.
 
-```{r}
-# add the sample metadata to the coldata 
+```{r sample metadata}
+# print out sample metadata
+head(sample_metadata)
+```
+
+Looking at this sample table, we see a column named `subdiagnosis` which accounts for the RMS subtype, ARMS or ERMS.
+We also see other columns that contain information about each specific sample and may contribute to differences seen across the samples.
+
+We can incorporate the information in this sample metadata table into the `colData` of the integrated SCE object.
+This will allow us to match each of the samples in the SCE object with the RMS subtype and also allow us to use any of the columns in the sample metadata for plotting.
+
+```{r modify coldata}
+# add the sample metadata to the colData from the integrated SCE object
 coldata_df <- colData(integrated_sce) %>%
+  # convert from DataFrame to data.frame
```

Comment:
We will have done this a few time by this point, so we might not need this comment, but it is fine to leave it in!